client/cypress/integration/dashboard/dashboard_spec.js

```diff
@@ -398,4 +398,70 @@ describe('Dashboard', () => {
       });
     });
   });
+
+  describe('switches to one column mode when viewport width is at 800px', () => {
+    before(function () {
+      cy.login();
+      createNewDashboardByAPI('Foo Bar')
+        .then(({ slug, id }) => {
+          this.dashboardUrl = `/dashboard/${slug}`;
+          return addTextboxByAPI('Hello World!', id);
+        })
+        .then((elTestId) => {
+          cy.visit(this.dashboardUrl);
+          cy.getByTestId(elTestId).as('textboxEl');
+        });
+    });
+
+    beforeEach(function () {
+      cy.visit(this.dashboardUrl);
+    });
+
+    it('widgets are full widths', () => {
+      cy.viewport(801, 800);
+      cy.get('@textboxEl').should(($el) => {
+        expect($el.width()).to.eq(378);
+      });
+      cy.viewport(800, 800);
+      cy.get('@textboxEl').should(($el) => {
+        expect($el.width()).to.eq(770);
+      });
+    });
+
+    it('hides edit option', () => {
+      cy.viewport(801, 800);
+      cy.getByTestId('DashboardMoreMenu')
+        .click()
+        .within(() => {
+          cy.get('li')
+            .contains('Edit')
+            .as('editButton')
+            .should('exist');
+        });
+
+      cy.viewport(800, 800);
+      cy.get('@editButton').should('not.be', 'visible');
+    });
+
+    it('disables edit mode', () => {
+      cy.viewport(801, 800);
+      editDashboard();
+      cy.get('button')
+        .contains('Apply Changes')
+        .as('saveButton')
+        .its('disabled')
+        .should('be', false);
```

Comment:
This doesn't seem to work, you can do it like this:

```javascript
cy.get('button')
  .contains('Apply Changes')
  .as('saveButton')
  .should('not.be.disabled');
```

Tip: When Cypress is asserting stuff it will make it explicit in the GUI:
![image](https://user-images.githubusercontent.com/3356951/55253293-b8dfdf80-5233-11e9-9762-e32d6f7de25e.png)